src/internal/functions/Get-ReadmeContent.ps1

```diff
@@ -142,7 +142,7 @@ You'll need to [file an issue]($PowerShellModuleProjectUri) with the upstream ma
 
 Problems with the type definition are when a value that should be valid according to the DSC resource's documentation and code is not accepted by the Puppet wrapper. If and when you run across one of these, please [file an issue]($BuilderModuleRepository/issues/new/choose) with the Puppet DSC Builder; this is where the conversion happens and once we know of a problem we can fix it and regenerate the Puppet modules. To help us identify the issue, please specify the DSC module, version, resource, property and values that are giving you issues. Once a fix is available we will regenerate and release updated versions of this Puppet wrapper.
 
-Problems with calling the underlying DSC resource become apparent by comparing `<value passed in in puppet>` with `<value received by DSC>`.
+Problems with calling the underlying DSC resource become apparent by comparing \`<value passed in in puppet>\` with \`<value received by DSC>\`.
```

Comment:
Aha! I think the problem is we were trying to escape the backticks in _markdown_ instead of in the PowerShell string - a backtick is a PowerShell string escape character, so we want it to end up in the markdown.
```suggestion
Problems with calling the underlying DSC resource become apparent by comparing ``<value passed in in puppet>`` with ``<value received by DSC>``.
```

That gives us a markdown block like this:

```markdown
Problems with calling the underlying DSC resource become apparent by comparing `<value passed in in puppet>` with `<value received by DSC>`.
```

Which should display like this:

> Problems with calling the underlying DSC resource become apparent by comparing `<value passed in in puppet>` with `<value received by DSC>`.